can't install themes

When trying to install a theme it complains about spawning `npm`, can it be added to the next release? Perhaps a persisting themes folder would need to be mapped.

```
root@helion:~# docker exec -it the-lounge node /app/node_modules/thelounge/index.js --home /config install lounge-theme-solarized
2017-11-21 21:34:33 [WARN] --home is deprecated and will be removed in a future version.
2017-11-21 21:34:33 [WARN] Use the LOUNGE_HOME environment variable instead.
2017-11-21 23:34:33 [WARN] Referring to CSS files in the theme setting of /config/config.js is deprecated and will be removed in a future version.
2017-11-21 23:34:33 [INFO] Retrieving information about the package...
2017-11-21 23:34:33 [INFO] Installing lounge-theme-solarized...
2017-11-21 23:34:33 [ERROR] Error: spawn npm ENOENT
```

Created PR https://github.com/linuxserver/docker-thelounge/pull/14, just a heads up though the theme in your example does not have up to date metadata and does not work with thelounge : 

```
[ERROR] lounge-theme-solarized does not have The Lounge metadata.
```

Tested some other themes though and they work.

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM lsiobase/alpine:3.7
+FROM lsiobase/alpine:3.8
 
 # set version label
 ARG BUILD_DATE
@@ -7,16 +7,13 @@ LABEL build_version="Linuxserver.io version:- ${VERSION} Build-date:- ${BUILD_DA
 LABEL maintainer="sparklyballs"
 
 # environment settings
-ENV LOUNGE_HOME="/config" \
+ENV THELOUNGE_HOME="/config" \
 NPM_CONFIG_LOGLEVEL="info"
 
 RUN \
- echo "**** install build packages ****" && \
- apk add --no-cache --virtual=build-dependencies \
-	nodejs-npm && \
  echo "**** install runtime packages ****" && \
  apk add --no-cache \
-	nodejs && \
+	nodejs-npm && \
  echo "**** install shout-irc ****" && \
  mkdir -p \
 	/app && \
@@ -26,8 +23,6 @@ RUN \
  echo "**** ensure public true on startup aka no users ****" && \
  sed -i "s/public: false,/public: true,/g" /app/node_modules/thelounge/defaults/config.js && \
  echo "**** cleanup ****" && \
- apk del --purge \
-	build-dependencies && \
  rm -rf \
 	/root && \
  mkdir -p / \

--- a/README.md
+++ b/README.md
@@ -97,6 +97,7 @@ To setup user account(s)
 
 ## Versions
 
++ **20.08.18:** Rebase to alpine 3.8.
 + **06.01.18:** Rebase to alpine 3.7.
 + **26.05.17:** Rebase to alpine 3.6.
 + **06.02.17:** Rebase to alpine 3.5.

